offers/README.md

```diff
@@ -243,21 +245,43 @@ curl -X POST \
        }'
 ```
 
-As required parameters, you need to set an offer `name`, your `accountId` as the `publisherId`, the `brandId`, a `startDate`, the type of offer between `amount` or `discount`, an offer `value`, and the `countryCode` where the offer will be available. The `value` is a percentage if the offer `type` is `discount` or a fixed amount in the local currency of the `countryCode` specified.
+### Required parameters
+
+There are a number of required parameters for the Offer to be created: 
+* `name`: the name of your offer
+* `publisherId` this is your Fidel `accountId`
+* `brandId` of the brand presenting the offer
+* `startDate`of the offer.  Any time added to the startDate will be local to the location.
+* `countryCode` where the offer will be available. 
+* `type` : `name`. Valid names are `amount` or `discount`.  
+* `type`: `value` Numeric value of the discount.
```

Comment:
some of these have `:`, some don't. also, some have a period at the end of the line, some don't. can you make this more uniform please?